### PR TITLE
fix some translations

### PR DESCRIPTION
**Pull request by stardustichi**

### CODE REVIEW

File: src/data/uma-title-requirements.csv [hunk 1]
```diff
@@ -1,11 +1,11 @@
 "text", "translation"
-"天気[雨]で4回以上出走し、3勝する","Participate a race with rainy weather 4 times or more\nWin at least 3 of them"
-"天気[曇り]で4回以上出走し、3勝する","Participate a race with cloudy weather 4 times or more\nWin at least 3 of them"
-"天気[雪]で3回以上出走し、2勝する","Participate a race with snowy weather 3 times or more\nWin at least 2 of them"
-"GⅠに1回以上出走し、\nGⅠ未勝利のまま育成ランクC以上で育成を完了する","<size=22>Participate a GI race once or more without winning any single GI race\nComplete the training with C or higher rank</size>"
+"天気[雨]で4回以上出走し、3勝する","Participate a race with rainy weather at least 4 times\nWin 3 of them"
```
Comment (FabulousCupcake) [inline src/data/uma-title-requirements.csv:2]: What do you think if we write it as `≥4 times` to save space? 😂

File: src/data/uma-title-requirements.csv [hunk 2]
```diff
@@ -1,11 +1,11 @@
...
+"天気[曇り]で4回以上出走し、3勝する","Participate a race with cloudy weather at least 4 times\nWin 3 of them"
+"天気[雪]で3回以上出走し、2勝する","Participate a race with snowy weather at least 3 times\nWin 2 of them"
+"GⅠに1回以上出走し、\nGⅠ未勝利のまま育成ランクC以上で育成を完了する","<size=22>Participate a GI race at least once and without winning any single GI race\nComplete the training with C or higher rank</size>"
 "ファン数が550000人以上になる","Earn a total of 550000 or more fans"
-"10回以上レースに出走し、\n芝のレースのみに出走して育成を完了する","<size=22>Participate a race 10 times or more\nEvery race has to be turf\nComplete the training</size>"
-"掛かったレースで3勝する","While under effect of temptation debuff\nWin a race 3 times"
+"10回以上レースに出走し、\n芝のレースのみに出走して育成を完了する","<size=22>Participate a race at least 10 times and every race has to be turf\nComplete the training</size>"
+"掛かったレースで3勝する","Win a race under panicked (掛かり) state 3 times"
```
Comment (FabulousCupcake) [inline src/data/uma-title-requirements.csv:8]:
```suggestion
"掛かったレースで3勝する","Win a race under panicked (掛かり) state 3 times"
```
While actively still panicking while crossing the finish line or had panicked in the race and won regardless?

The sentence seem to say the former, but that seems hard to achieve and my gut says it's the latter 🤔

Comment (stardustichi) [reply src/data/uma-title-requirements.csv:8]: its the latter

File: src/data/uma-title-requirements.csv (hunk 1 above)
Comment (stardustichi) [reply src/data/uma-title-requirements.csv:2]: this is great idea, should we?

Comment (FabulousCupcake) [reply src/data/uma-title-requirements.csv:2]: Considering length limitations, I think it's actually good.  
Even if many text don't reach the limit yet, it still frees more room/space to breathe.

File: src/data/uma-title-requirements.csv (hunk 2 above)
Comment (stardustichi) [reply src/data/uma-title-requirements.csv:8]: if you have suggestion let me know

Comment (FabulousCupcake) [reply src/data/uma-title-requirements.csv:8]: Yes, I've been staring at this page for 5 minutes thinking about it 😂

Comment (stardustichi) [reply src/data/uma-title-requirements.csv:8]: lol i know right

Comment (FabulousCupcake) [reply src/data/uma-title-requirements.csv]: Current one from bb44f17 looks good 👍

File: src/data/uma-title-requirements.csv (hunk 1 above)
Comment (stardustichi) [reply src/data/uma-title-requirements.csv]: let me know if theres more that need to be changed

File: src/data/uma-title-requirements.csv [hunk 3]
```diff
@@ -1,139 +1,139 @@
...
-"育成でオークスを勝利したウマ娘から想いを継承し、\nオークス、天皇賞（秋）を勝利し、\nファン数が320000人以上になる","<size=22>Inherit memories from parents that won the Oaks\nWin the Oaks and Tenno Sho (Autumn)\nEarn a total of 320000 or more fans</size>"
-"重賞に23戦以上出走し、\nうち菊花賞、天皇賞（春）を2番人気以上で勝利し、\nファン数が320000人以上で育成を完了する","<size=22>Participate a big prize 23 times or more\nWin the Kikka Sho and Tenno Sho (Spring) being the 2nd or higher popular\nEarn a total of 320000 or more fans</size>"
-"1番人気で弥生賞、皐月賞、日本ダービー、菊花賞を勝利して、\n有馬記念（クラシック級）を勝利して、\nファン数が240000人以上になる","<size=22>Win the Yayoi Sho, Satsuki Sho, Japan Derby and Kikka Sho being the 1st popular\nWin the Arima Kinen in 2nd year\nEarn a total of 240000 or more fans</size>"
-"育成ランク[S]以上のウマ娘から想いを継承し、\n無敗でホープフルステークス、弥生賞、皐月賞を勝利し、\n基礎能力[スピード]が1200以上になる","<size=22>Inherit memories from parents with S or higher rank\nWin the Hopeful Stakes, Yayoi Sho, and Satsuki Sho while having zero losses\nReach 1200 or higher speed</size>"
+"スキルを15個以上獲得する","Learn ≥15 skills"
+"基礎能力[スピード]が800以上かつ\n芝のレースを15回以上出走する","Participate a turf race ≥15 times\nReach ≥800 speed"
+"重賞未勝利でGⅠを勝利する","Without winning a single big race\nWin a GI race"
```
Comment (FabulousCupcake) [inline src/data/uma-title-requirements.csv:39]: 🤔 this is unclear / seems machine-translated without proper context  
Title associated to this seems to be `台風の目` if that helps

Comment (FabulousCupcake) [reply src/data/uma-title-requirements.csv:39]: There are many other "big race" description; not sure what they exactly are but I'm guessing it's 18 participant races?
  
In any case seems like it'll be too long for some lines, so I'm merging this PR

**Review by FabulousCupcake — APPROVED** [review]

**Comment by FabulousCupcake** [comment]
Thank you!

**Pull request merged**